kalite/distributed/static/js/distributed/rating/views.js

```diff
@@ -203,19 +210,8 @@ var TextView = BaseView.extend({
         return this;
     },
 
-    text_changed: function() {
+    text_changed: _.throttle(function() {

```

Comment:
@jamalex I prefer `throttle` to `debounce` here -- delaying execution until _after_ input stops (especially for a long amount of time) leaves a window where the view/model could be changed (as with a page close or navigating to a different content item) before the model is synced.